deeplake/core/vectorstore/deeplake_vectorstore.py

```diff
@@ -343,6 +343,7 @@ def search(
 
         # if embedding_function is not None or embedding is not None:
         query_emb = dataset_utils.get_embedding(
+            query,
```

Comment:
@adolkhan Why is this necessary if we just return None? `TQL` query and `get_embedding` are fundamentally contradictory?

Reply:
Agree, haven't thought about it. Moved it to a higher level